Decred Wallet/Utils/Settings.swift

```diff
@@ -15,6 +15,7 @@ class Settings {
         static let SpendingPassphraseSecurityType = "spending_security_type"
         static let DefaultWallet = "default_wallet"
         static let HiddenWalletPrefix = "hidden"
+        static let newWalletSetUp = "new_wallet_set_up"
```

Comment:
2 things
- Let's move this property to the top of the `Keys` struct, the keys are sort of arranged in the order we presume they'd be used to make scanning through easier.
- We should maintain a naming convention for these keys, all other keys start with a capital letter, can do the same for this `newWalletSetUp` key.